src/cljs/main/broadfcui/common/method/config_io.cljs

```diff
@@ -78,18 +97,26 @@
                        {:on-upload
                         (fn [{:keys [file-contents]}]
                           (let [uploaded-inputs (utils/parse-json-string file-contents true)
-                                new-inputs (merge (:inputs @state) uploaded-inputs)]
+                                new-inputs (merge (:inputs @state) (into {} (map (fn [[k v]] [k (utils/->json-string v)]) uploaded-inputs)))]
                             (when-not editing? (begin-editing))
-                            (after-update #(doseq [[k v] (vec new-inputs)] ((k @locals) :set-value v)))
+                            (after-update #(doseq [[k v] (vec new-inputs)] (when (contains? @locals k) ((k @locals) :set-value v))))
```

Comment:
Added these two lines for now, these are in Brian's PR, will rebase off of his when it's merged. This PR should wait on that one.